lambda/smsUtil.js

```diff
@@ -32,4 +33,16 @@ exports.sendSMSMessage = async function (payload, cancellationLink){
       logger.error(e)
       return sendResponse(400, { msg: 'SMS notification failed.', title: 'Operation Failed' });
     }
+  }
+
+  function convertPassType(passType){
+    const passTimeOptions = ["am","pm","all-day"];
+    if(passType === "AM"){
+      return passTimeOptions[0];
+    }else if(passType === "PM"){
+      return passTimeOptions[1];
+    }else if (passType === "DAY"){
```

Comment:
Spacing of `{` and `}` braces needs to be updated.

👍 / that nit - as @cameronpettit mentioned maybe better with `switch` statement.

Reply:
Changed to switch statement which fixed spacing in: [2cb314f](https://github.com/bcgov/parks-reso-api/pull/335/commits/2cb314f9401f64b812bad4da6d71baddee166dfe)